src/settings/config.rs

```diff
@@ -75,13 +74,14 @@ impl Config {
         }
     }
 
-    fn load_from_path(path: &Path) -> Result<Self, String> {
+    fn load_from_path(path: &Path) -> Result<Self, Option<String>> {
         let toml = std::fs::read_to_string(path).map_err(|e| {
-            format!(
+            log::debug!(
```

Comment:
If you take a look at `src/main.rs`, you'll see that config is being loaded before logger is initialized, so this message won't be logged.

Reply:
Nice catch, instead of this added path.exists() check.